Chess/Assets/Scripts/ArrayChess/PieceMover.cs

```diff
@@ -162,63 +128,45 @@ void Update()
     //        cacheColor = parentSquareRend.material.color;
     //        Debug.Log(cacheColor);
 
-
     //        isCached = true;
-
-
     //    }
-
     //}
 
 
-
-    void resetColors(bool updateColor)
```

Comment:
Looking back at this I think this was the other unecessary abstraction, so I put it back. We could get this working and then decide if its repeating itself a lot later on